### PR TITLE
[react-data-grid] Add getCellActions to GridProps

### PR DESCRIPTION
**Pull request by JCGrant**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: http://adazzle.github.io/react-data-grid/examples.html#/cell-actions
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/react-data-grid/index.d.ts [hunk 1]
```diff
@@ -219,6 +219,22 @@ declare namespace AdazzleReactDataGrid {
          * @param columnKey the column key that we are looking to pull values from
          */
         getValidFilterValues?: (columnKey: string) => Array<any>
+
+        getCellActions?: (column: Column, row: object) => (ActionButton | ActionMenu)[]
+    }
+
+    type ActionButton = {
+        icon: string;
+        callback: Function;
```
Comment (SupernaviX) [inline types/react-data-grid/index.d.ts]: Looks good, but could you define the callbacks as `() => void` instead of Function?

Comment (JCGrant) [reply types/react-data-grid/index.d.ts]: Done!

Comment (SupernaviX) [reply types/react-data-grid/index.d.ts]: The one on line 228 as well, please

Comment (JCGrant) [reply types/react-data-grid/index.d.ts]: Also done!

**Review by SupernaviX — APPROVED** [review]

**Pull request merged**